reduction_scripts/reduce_data.py

```diff
@@ -686,16 +686,14 @@ def run_cube_gen(metadata, prev_suffix, curr_suffix, **args):
                     ds2 = (t2 - t1).total_seconds()
                     if ds1>0 and ds2>0:
                         # Alright, I need to interpolate betweent the two arcs
-                        file_arm = sci_header['ARM']
-                        if file_arm == 'Red':
+                        file_camera = sci_header['CAMERA']
+                        if file_camera == 'WiFeSRed':
```

Comment:
Updated this again to avoid issues with headers by looking at the keyword in `'CAMERA'` instead of `'ARM'` because `'CAMERA'` is the header keyword used in `classify` at the beginning of the script so should be all good with this data. Otherwise, the pipeline would crash in `classify` as we always rely on good data in the file headers (to fix that is a problem of the telescope/camera, not the pipeline).

Reply:
Yup, nice work, good work on the same assumptions as the classifier.